apps/documentation/.gitignore

```diff
@@ -1,5 +1,4 @@
 # next.js
-/.next/
```

Comment:
changed the output directory to `build/` to be inline with our standard build patterns